README.md

```diff
@@ -41,6 +41,18 @@ Source-build solves common challenges that most developers encounter when trying
 
 Starting with .NET 6, the core source-build infrastructure is integrated into the [dotnet/installer](https://github.com/dotnet/installer/tree/main/src/SourceBuild) repo. The `main` branch on this repo now contains the tooling needed to build .NET's external dependencies from source.
 
+## .NET in Linux Distribution
+| Distro |Package Feed|Maintainer|
+|---|---|---|
+| Alpine | [Community](https://pkgs.alpinelinux.org/packages?name=dotnet*&branch=v3.16&repo=&arch=&maintainer=) | [@ayakael](https://github.com/ayakael) |
```

Comment:
Is the maintainer link required?  I was expecting the @ syntax would just work in GH.  If not, should the @ be droppedn?

Reply:
I'm sorry, it looks like I was wrong.  The hyperlink is required.  You will need to use `[@ayakael](https://github.com/ayakael)` and I think the @ is usage is appropriate.